doc/docs/master/deploy-manage/deploy/deploy-pachyderm-jupyterhub.md

```diff
@@ -30,124 +39,69 @@ deployed on the same Kubernetes cluster. You deploy Pachyderm by
-
-You can enable Pachyderm access controls to use Pachyderm credentials
-to log in to JupyterHub. However, this
-is not required, and if you prefer to use JupyterHub authentication
-instead, skip this section.
+Before deploying Pachyderm IDE, configure the following prerequisites:
 
-Pachyderm Access Controls is an enterprise feature. You need to have
-an active Enterprise token to enable it.
+* Deploy Pachyderm 1.11.0 or later as described in
+[Deploy Pachyderm](https://docs.pachyderm.com/latest/deploy-manage/deploy/)
+on a supported Kubernetes platforms:
 
-To enable Pachyderm authentication, follow the steps in
-[Configure Access Controls](https://docs.pachyderm.com/latest/enterprise/auth/auth/)
+  - Google Kubernetes Engine (GKE) with Kubernetes v1.13
+  - Amazon Elastic Container Service (EKS) with Kubernetes v1.13
+  - Docker Desktop for Mac with Kubernetes v1.14
 
-!!! note
-    If you decide to enable Pachyderm authentication later, you
-    will need to either manually reconfigure or redeploy JupyterHub.
-    Your existing notebooks will be preserved after
-    the redeploy.
-
-### Install Helm
-
-Helm is a package manager for Kubernetes that controls the JupyterHub
-installation. Our deployment script requires Helm 3 or later.
-
-To install Helm on your client machine, follow the steps in the
-[Helm Documentation](https://helm.sh/docs/intro/install/).
-
-## Deploy JupyterHub
+    For more information about JupyterHub requirements for Kubernetes,
+    see [Zero to JupyterHub with Kubernetes](https://zero-to-jupyterhub.readthedocs.io/en/latest/).
 
-After you deploy Pachyderm in one of the supported environments and
-install Helm on your client machine, you can use the deployment script
-to deploy JupyterHub.
+* Register your Enterprise token as described in
+[Activate Pachyderm Enterprise Edition](https://docs.pachyderm.com/latest/enterprise/deployment/#activate-pachyderm-enterprise-edition).
 
-To deploy JupyterHub, complete the following steps:
+* Enable [Pachyderm Access Controls](https://docs.pachyderm.com/latest/enterprise/auth/auth/).
 
-<!--1. Clone the [jupyterhub-pachyderm](https://github.com/pachyderm/jupyterhub-pachyderm)
-repository:
+## Deploy Pachyderm IDE
 
-   ```bash
-   git clone git@github.com:pachyderm/jupyterhub-pachyderm.git
-   ```
+After you deploy Pachyderm and enable authentication,
+deploy the Pachyderm IDE by running:
 
-1. Change the current directory to `jupyterhub-pachyderm/`:
+```bash
+pachctl deploy ide
+```
 
-   ```bash
-   cd jupyterhub-pachyderm
-   ```
--->
-1. Run the `./init.py` script:
+Pachyderm deploys and configures JupyterHub and JupyterLab, which
+might take some time. 
 
-   ```bash
-   ./init.py
-   ```
+## Log in to Pachyderm IDE
 
-   **System response:**
+After you deploy the Pachyderm IDE, you can access the JupyterLab UI
+in a web browser through the Pachyderm cluster hostname on port
+`80`. To get your the IP address of the Pachyderm IDE,
+run the following command:
```

Comment:
1) Nit: you access the JupyterHub UI first
2) This is only true for local deployments, for cloud deployments it sets up a separate service with a dedicated IP

Reply:
Not sure what you want to change there. It seems that what we have below could be correct?